src/libOpenImageIO/imagebufalgo_draw.cpp

```diff
@@ -1162,8 +1164,14 @@ ImageBufAlgo::render_text(ImageBuf& R, int x, int y, string_view text,
         float val   = t[0];
         float alpha = a[0] * textalpha;
         R.getpixel(r.x(), r.y(), pixelcolor);
-        for (int c = 0; c < nchannels; ++c)
-            pixelcolor[c] = val * textcolor[c] + (1.0f - alpha) * pixelcolor[c];
+        if (alpha == 0.0)
+            continue;
+        for (int c = 0; c < nchannels; ++c) {
+            if (c == alpha_channel)
+                pixelcolor[c] = alpha + (1.0f - alpha) * pixelcolor[c];
+            else 
+                pixelcolor[c] = (val * alpha * textcolor[c]) + (1.0f - alpha) * pixelcolor[c];
```

Comment:
You may have noticed CI failures. Basically, every test with text in it is failing with this PR.

I think this line is the part that's responsible. Compared to the old code, you have an extra multiplication of the foreground color by alpha. I believe (but admit that I am not 100% sure) that it was correct before because freetype is supplying color results that are already scaled by the glyph's alpha coverage, so now you are *doubly* accounting for alpha coverage in the foreground.